pkg/registry/tls_test.go

```diff
@@ -0,0 +1,35 @@
+package registry_test
```

Comment:
looks like we're missing copyright headers on this entire package actually

Reply:
Added them to the entire package.